CMakePresets.json

```
@@ -26,7 +26,7 @@
     },
     {
       "name": "imcs_workstation_RELWITHDEBINFO",
-      "displayName": "Release build for an IMCS workstation",
+      "displayName": "Debug build for an IMCS workstation",
```

@RShaw026 Please update with the following suggestion:
```suggestion
      "displayName": "Release build with debug information for an IMCS workstation",
```
I have missed that in my original PR. Sorry.